newsfragments/2223.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fixed clicking on a file link in some instances
```

Comment:
This newsfragments don't seem related to this PR...

Reply:
hum, I'll remove them